app.py

```diff
@@ -336,13 +338,18 @@ def generate_graph(street, direction, graph_name, day_type='Weekday', period='AM
               href = '/css/style.css'),
           
 html.Div(children=[html.H1(children=TITLE, id='title'),
+                   html.Button('info', id = 'info', className = 'info')
```

Comment:
This info button is a little subtle, black text on dark blue background is hard to see